reconciletags/reconciletags.py

```diff
@@ -46,19 +46,22 @@ def add_tags(self, digest, tag, dry_run):
     # This turns a list of lists into one flat list of tags
     def flatten_tags_list(self, list_of_lists):
         flat_tags_list = []
+        latest = ''
         for sublist in list_of_lists:
-            for tag in sublist:
+            for tag in sublist['tags']:
                 if tag:
+                    if tag == 'latest':
+                        latest = sublist['digest'][len('sha256:'):]
                     flat_tags_list.append(tag)
-        return flat_tags_list
+        return flat_tags_list, latest
 
     def get_existing_tags(self, repo):
         output = json.loads(self.call('gcloud beta container images list-tags '
                             '--no-show-occurrences {0}'.format(repo), False))
 
-        list_of_tags = [image['tags'] for image in output]
-        existing_tags = self.flatten_tags_list(list_of_tags)
-        return existing_tags
+        list_of_tags = [image for image in output]
```

Comment:
This would be an extra call, but I think all this code would be easier to follow if we fetch the latest tag separately. WDYT?